common/src/main/res/values/strings.xml

```diff
@@ -1041,4 +1041,6 @@
     <string name="not_registered">Please launch the Home Assistant app and login before you can use the assist feature.</string>
     <string name="ha_assist">HA: Assist</string>
     <string name="only_favorites">Only Show Favorites</string>
+    <string name="beacon_scanning">Beacon Scanning</string>
```

Comment:
To match the sensor name:
```suggestion
    <string name="beacon_scanning">Beacon Monitor Scanning</string>
```

Reply:
and stop_scanning maybe only stop/disable same as high accurancy action notification?

![obrazek](https://user-images.githubusercontent.com/57643470/222118992-ef6ee032-283a-4c71-be45-352a6a76f8c8.png)

in case of translation it can be a string 🤔

Reply:
There is already an action like that

Reply:
I think @leroyloren was suggesting changing the action to 'Stop' or 'Disable', so a more generic string that may already be translated and short like other notifications with actions posted by the app. 

(Existing actions for other notifications are 'Disable' or 'Settings')